docker-compose-koppie.yml

```diff
@@ -33,82 +32,67 @@ services:
       - 1049:1048
     volumes:
       - ./bin:/app/samvera
-      - .koppie:/app/samvera/hyrax-webapp
+      - .koppie:/app/samvera/hyrax-koppie
```

Comment:
can I ask what we accomplish by making this different? its extra mental overhead to remember which is which unless it serves some purpose.

Reply:
This was done to better support having both .dassie and .koppie available in the hyrax-engine-dev container image, but in the end it might not be necessary. My thinking was it is easier to know which test app you're in by the directory name, but you make a good point too. I'll look into undoing the hyrax-koppie dir.